### PR TITLE
Create ClearDotTax.md

### PR DESCRIPTION
**Pull request by rajgoesout**
#### Project Abstract

Calculating and filing taxes for Polkadot/Kusama and its parachains is a strenuous and time-consuming process. Existing solutions don't fully solve it, and offer a poor user experience. ClearDotTax aims to give users their Polkadot tax report for their jurisdiction in one click.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (Polkadot AssetHub (DOT, USDC & USDT) address in the application and bank details via email, if applicable).
- [x] I understand that 30% of each milestone will be paid in vested DOT, to the Polkadot address listed in the application.
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by rajgoesout** [comment]
I have read and hereby sign the Contributor License Agreement.

File: applications/ClearDotTax.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+- https://twitter.com/0xDeep
+- https://twitter.com/axshay613
+
+### Project Details
+
+#### Key Deliverables
+
+- ##### Backend API
+  To fetch users' transactions across Polkadot and Kusama relay chains; and all major parachains of Polkadot & Kusama (ones supported by Subscan).
+- ##### [Transactions Dashboard](#milestone-1--transactions-dashboard)
```
Comment (semuelle) [inline applications/ClearDotTax.md:123]: We usually expect some wireframes/designs for any frontend deliverables. Do you have anything you can share here?

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the application, @rajgoesout. If I'm understanding correctly, you are planning to run the project as a service and charging for subscriptions, correct? 

First, as you know, the grants program expects all deliverables to be open source, including dependencies. How are you ensuring that people will pay for your service?

Second, have you considered applying for a [Decentralized Futures](https://futures.web3.foundation/) grant? This is exactly the kind of project the program is built for. And applications up to 50k USD can be fast-tracked.

File: applications/ClearDotTax.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      a. Native DOT<br>
+      b. Manta<br>
+      c. Astar<br>
+      d. Hydradx<br>
+      e. Acala & Karura
+  3.  Yield Farms<br>
+      a. Stellaswap
+  4.  Liquidity Pools<br>
+      a. Stellaswap<br>
+      b. Acala & Karura
```
Comment (PieWol) [inline applications/ClearDotTax.md:141]: please don't use any unclosed html tags. We need all applications to compile with mdx-5.
Give your application a check here. https://mdxjs.com/playground/

**Comment by rajgoesout** [comment]
Hey @semuelle thanks for the review. Answering inline:

> Thanks for the application, @rajgoesout. If I'm understanding correctly, you are planning to run the project as a service and charging for subscriptions, correct?

Yes.

> First, as you know, the grants program expects all deliverables to be open source, including dependencies. How are you ensuring that people will pay for your service?

The way we were thinking was: launch a PoC which is open source with the help of this grant; and later on add support for more protocols/features via new modules, which could be closed source. This is based on our experience with [YieldScan](https://github.com/yieldscan), which despite being completely open source, had paid users; given people don't want the hassle of running their own infra or updating the integrations when protocols update their APIs/code.

> Second, have you considered applying for a Decentralized Futures grant? This is exactly the kind of project the program is built for. And applications up to 50k USD can be fast-tracked.

We weren't aware of this program before. What we can do now:
- apply for the Decentralized Futures grant with an updated scope
- apply for both, this open source w3f grant and the Decentralized Futures program with separate deliverables

Happy to follow whichever course of action the foundation thinks would be more beneficial for the ecosystem.

File: applications/ClearDotTax.md (hunk 1 above)
Comment (rajgoesout) [reply applications/ClearDotTax.md:123]: Got it. We're finalizing our designs - will attach them and ping on this thread as soon it's ready (should take about 3 days).

**Comment by semuelle** [comment]
> What we can do now:
> 
>     * apply for the Decentralized Futures grant with an updated scope
> 
>     * apply for both, this open source w3f grant and the Decentralized Futures program with separate deliverables

@rajgoesout, please apply for a DF grant first, as the program ends in June. We can always come back to the grants program if needed.

**Comment by rajgoesout** [comment]
@semuelle Thank you for the advice. Just submitted the DF application with a revised scope.

**Comment by semuelle** [comment]
Thanks, @rajgoesout. I'll close this PR for now then.

**Pull request closed without merging**